src/content/advanced/tls-certificates/index.md

```diff
@@ -0,0 +1,104 @@
+---
+linkTitle: TLS certificates with cert-manager
+title: Obtaining TLS certificates for Ingresses with cert-manager
+description: Configure cert-manager to automatically obtain TLS certificates for Ingresses
+weight: 20
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-cabbage
+user_questions:
+  - How do I install cert-manager?
+  - How do I obtain TLS certificates?
+last_review_date: 2022-07-19
+---
+
+Exposing HTTP services using an Ingress is a pretty straightforward task in Kubernetes. Using plain HTTP is, however, discouraged in favor of using HTTPS. `cert-manager` provides a simple, declarative way to automatically obtain TLS certificates for the endpoints we expose.
+
+Our App Platform provides a [`cert-manager-app`](https://github.com/giantswarm/cert-manager-app) that users can install. In order to configure it for TLS certificate management, we should install a `ClusterIssuer` that solves DNS challenges. Note that this also works for internal Load Balancers.
```

Comment:
The `ClusterIssuer` and `Issuer` CRD comes with `cert-manager`. `cert-manager` creates a `ClusterIsser` using the HTTP challenge by default. This works fine in most cases.

Disabling the creation of this default issuer and creating an own one is probably only require for pure internal environments or for issuing wildcard certificates. So normally you don't need to disable the creation of the default issuer.

Maybe you could point this out here. :)